README.md

```diff
@@ -1,2 +1,37 @@
-# Blog-App
-Thegit
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Blog App
+
+> The Blog app will be a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## Built With
+
+- Ruby on Rails
+- HTML & CSS
+- GitFlow
+- Linters
+- Documented Work
+
+
```

Comment:
## [OPTIONAL] 
* I strongly suggest you add setup instructions to your readme file so that visitors would know how to properly install and run your application locally